library/src/main/java/com/prolificinteractive/materialcalendarview/format/WeekDayFormatter.java

```diff
@@ -17,5 +17,5 @@
     /**
      * Default implementation used by {@linkplain com.prolificinteractive.materialcalendarview.MaterialCalendarView}
      */
-    public static final WeekDayFormatter DEFAULT = new CalendarWeekDayFormatter(CalendarUtils.getInstance());
+    WeekDayFormatter DEFAULT = new CalendarWeekDayFormatter(CalendarUtils.getInstance());
```

Comment:
Missing another one in `com/prolificinteractive/materialcalendarview/format/DayFormatter.java`

Reply:
Done